hello-world/README.md

```diff
@@ -422,6 +422,120 @@ dapr stop --app-id pythonapp
 
 To see that services have stopped running, run `dapr list`, noting that your services no longer appears!
 
+## [Optional Steps] VS Code Debugging
+
+Leveraging VS Code you can manually configure your launch.json and task.json to launch Dapr and your applications into debug mode.
+
+### [Optional] Step 1 
+Find the .vscode folder present in your workspace and make sure that a launch.json and tasks.json file are present.
+If not please create one through the VS Code debugger, by following the guidelines the debugger displays when clicked on.
+
+![VSCode Debugger Screenshot](./img/launch-configuration.png)
+
+VS Code will try to automatically detect your debug environment, but if this fails, you will have to choose it manually:
+
+![VSCode Debug Environment Screenshot](./img/debug-environments.png)
+
+If you go back to the File Explorer view (⇧⌘E), you'll see that VS Code has created a .vscode folder and added the launch.json file to your workspace.
+
+### [Optional] Step 2 
+
+Once those files are created copy and paste the following configurations:
+
+#### Launch Configs
+```json
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "pwa-node",
+            "request": "launch",
+            "name": "Nodeapp with Dapr",
+            "skipFiles": [
+                "<node_internals>/**"
+            ],
+            "program": "${workspaceFolder}/app.js",
+            "preLaunchTask": "daprd-debug-node",
+            "postDebugTask": "daprd-down-node"
+        },
+        {
+            "type": "python",
+            "request": "launch",
+            "name": "Pythonapp with Dapr",
+            "program": "${workspaceFolder}/app.py",   
+            "console": "integratedTerminal",
+            "preLaunchTask": "daprd-debug-python",
+            "postDebugTask": "daprd-down-python"
+        }
+    ],
+    "compounds": [
+        {
+            "name": "Node/Python Dapr",
+            "configurations": ["Nodeapp with Dapr","Pythonapp with Dapr"]
+                }
+    ]
+}
+```
+
+#### Task Config
+
+```json
+{
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"appId": "nodeapp",
+			"appPort": 3000,
+			"httpPort": 3500,
+			"metricsPort": 9090,
+			"label": "daprd-debug-node",
+			"type": "daprd"
+		},
+		{
+			"appId": "nodeapp",
+			"label": "daprd-down-node",
+			"type": "daprd-down"
+		},
+		{
+			"appId": "pythonapp",
+			"httpPort": 53109,
+			"grpcPort": 53317,
+			"metricsPort": 9091,
+			"label": "daprd-debug-python",
+			"type": "daprd"
+		},
+		{
+			"appId": "pythonapp",
+			"label": "daprd-down-python",
+			"type": "daprd-down"
+		}
+	]
+}
+```
+
+The general understanding of what these parameters and what the task.json and launch.json represent in VS Code can be found in the Dapr doc[How-to: VSCode Multiple Dapr Apps Debugging](https://docs.dapr.io/developing-applications/ides/vscode/vscode-how-to-debug-multiple-dapr-apps/)
```

Comment:
This link is a 404: https://docs.dapr.io/developing-applications/ides/vscode/vscode-how-to-debug-multiple-dapr-apps/

Reply:
Oh, you already address this above, nvm.